addons/web/static/src/js/database_manager/database_manager.js

```diff
@@ -0,0 +1,50 @@
+$(function() {
```

Comment:
I've two concerns about this file. First, it would be nice if it could stop using jquery. Second, I don't know we it should be located (not even sure it should be in /web). @ged-odoo What do you think?

Reply:
Just to be sure you know, I copy pasted the code without modification that was previously on `database_manager.html`.

Reply:
Yep I know.

Reply:
Can you move the file in web/static/src/public (you'll need to create the folder) ?

Reply:
Done